samples/contrib/aws-samples/ground_truth_pipeline_demo/README.md

```diff
@@ -34,25 +34,92 @@ client_ID = App client
+   ```
+   aws eks describe-cluster --name <cluster_name> --query "cluster.identity.oidc.issuer" --output text
+   ```
+3. Create a file named trust.json with the following content.   
+   Replace `<OIDC_URL>` with your OIDC issuer URL **(Don’t include https://)** and `<AWS_account_number>` with your AWS account number. 
+   ```
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Effect": "Allow",
+         "Principal": {
+           "Federated": "arn:aws:iam::<AWS_account_number>:oidc-provider/<OIDC_URL>"
+         },
+         "Action": "sts:AssumeRoleWithWebIdentity",
+         "Condition": {
+           "StringEquals": {
+             "<OIDC_URL>:aud": "sts.amazonaws.com",
+             "<OIDC_URL>:sub": "system:serviceaccount:kubeflow:pipeline-runner"
+           }
+         }
+       }
+     ]
+   }
+   ```
+4. Create an IAM role using trust.json. Make a note of the ARN returned in the output.
+   ```
+   aws iam create-role --role-name kfp-example-pod-role --assume-role-policy-document file://trust.json
+   aws iam attach-role-policy --role-name kfp-example-pod-role --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+   aws iam get-role --role-name kfp-example-pod-role --output text --query 'Role.Arn'
+   ```
+5. Edit your pipeline-runner service account.
+   ```
+   kubectl edit -n kubeflow serviceaccount pipeline-runner
+   ```
+   Add `eks.amazonaws.com/role-arn: <role_arn>` to annotations, then save the file. Example:   
+   ```
+   apiVersion: v1
+   kind: ServiceAccount
+   metadata:
+     annotations:
+       eks.amazonaws.com/role-arn: <role_arn>
+       kubectl.kubernetes.io/last-applied-configuration: |
+         {"apiVersion":"v1","kind":"ServiceAccount","metadata":{"annotations":{},"labels":{"app":"pipeline-runner","app.kubernetes.io/component":"pipelines-runner","app.kubernetes.io/instance":"pipelines-runner-0.2.0","app.kubernetes.io/managed-by":"kfctl","app.kubernetes.io/name":"pipelines-runner","app.kubernetes.io/part-of":"kubeflow","app.kubernetes.io/version":"0.2.0"},"name":"pipeline-runner","namespace":"kubeflow"}}
+     creationTimestamp: "2020-04-16T05:48:06Z"
+     labels:
+       app: pipeline-runner
+       app.kubernetes.io/component: pipelines-runner
+       app.kubernetes.io/instance: pipelines-runner-0.2.0
+       app.kubernetes.io/managed-by: kfctl
+       app.kubernetes.io/name: pipelines-runner
+       app.kubernetes.io/part-of: kubeflow
+       app.kubernetes.io/version: 0.2.0
+     name: pipeline-runner
+     namespace: kubeflow
+     resourceVersion: "11787"
+     selfLink: /api/v1/namespaces/kubeflow/serviceaccounts/pipeline-runner
+     uid: d86234bd-7fa5-11ea-a8f2-02934be6dc88
+   secrets:
+   - name: pipeline-runner-token-dkjrk
+   ```
+**Role 2]** For sagemaker job to access S3 buckets and other Sagemaker services. This Role ARN is given as an input to the components.
```

Comment:
What is this role used for to launch training jobs ? 

Can we just say say create a role with  AWS managed policy https://docs.aws.amazon.com/sagemaker/latest/dg/access-policy-aws-managed-policies.html for sagemaker ?

Reply:
AWS managed policy doesn’t have s3 access